nav2_bt_navigator/README.md

```diff
@@ -46,7 +46,20 @@ The BT Navigator package has two sample XML-based descriptions of BTs.  These tr
 </root>
 ```
 
-Navigate with replanning is composed of the following custom decorator, condition and action nodes:
+Navigate with replanning is composed of the following custom decorator, condition,
+contro and action nodes:
+
+#### Control Nodes
+* FirstCompletion: This is a variant of a Parallel Node. When this node is ticked,
```

Comment:
The description sounds like "hey lets a few trees run in parallel" but the name sounds like "checks if something has been completed one first time as a binary gate", maybe one of these names should change? I'm not entirely sure that's happening here either from just reading the code

Reply:
Yeah. I'm not thrilled with the name, either. Basically, this node runs all it's children in parallel until one of them returns a SUCCESS or FAILURE. It then stops them all and returns that "First Completion" status.